recipes-ccsp/ccsp/rdkgponmanager.bb

```diff
@@ -0,0 +1,71 @@
+SUMMARY = "RDK GPON Manager component"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
+
+DEPENDS = "ccsp-common-library dbus rdk-logger utopia hal-platform json-hal-lib"
+
+require ccsp_common.inc
```

Comment:
Please change this to
`require recipes-ccsp/ccsp/ccsp_common.inc`
to resolve recipe parse error.

Reply:
Addressed. Can you pls review?